api/db/changelog/db.changelog-127-verified-institutional-affiliation.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="dchangelog-127-verified-institutional-affiliation">
+
+    <createTable tableName="verified_user_institution">
```

Comment:
The Swagger-API model class for this is `VerifiedInstitutionalAffiliation` so it might be better for this to match: `verified_institutional_affiliation`